### DISCUSSION TITLE
网络出错：7月2日突然无法搜索、登陆，不论是扫码，还是帐号都不行

### DISCUSSION THREAD

**Discussion by jkqxl** [ywmoyue/biliuwp-lite#692] [Q&A]
### Bug 描述

如标题，昨天还是好好的，今日突然就掉了，而且也登不上去了。

测试，搜索也有问题

版本：BiliLite.Packages_4.6.33.0_x64.zip

### 操作步骤

```plain text
1. 打开应用
2. 出现网络错误的截图
3. 重新登录无效
```

### 期望结果

或许不只是个人问题，也或许是 B站官方服务器抽风，等下再看看。

### 截图

![Snipaste_2024-07-02_11-00-22](https://github.com/ywmoyue/biliuwp-lite/assets/72729140/13dce4ed-e11e-44ce-a33e-8e31b4d5c186)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

Windows 10 21H1 (Build 19043)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
@jkqxl 麻烦查看软件帮助页说明检查本机环境问题

![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/41803c81-d10e-448f-8e79-bebaf140d737)

**Reply by ywmoyue** [reply]
@jkqxl 问题是否解决？
